bucket/MPC-Video-Renderer.json

```diff
@@ -0,0 +1,17 @@
+{
+    "version": "0.6.7",
+    "description": "A free and open-source video renderer for DirectShow.",
+    "homepage": "https://github.com/Aleksoid1978/VideoRenderer",
+    "license": "GPL-3.0-or-later",
+    "suggest": {
+        "Media Player Classic - Home Cinema fork": "extras/mpc-hc-fork"

```

Comment:
```suggestion
        "MPC-BE": "extras/mpc-be"
```

> ... but full support is available only in the MPC-BE. Recommended MPC-BE 1.6.6 or newer.
>
> https://github.com/Aleksoid1978/VideoRenderer/blob/master/Readme.md

Reply:
yah right but can we add both like mpc-hc-frok and mpc-be Suggestion
https://github.com/clsid2/mpc-hc/blob/develop/Readme.md?plain=1#L55